class task.py

```diff
@@ -0,0 +1,39 @@
+class DNA:
+    def __init__(self, seqq):
+        self.seq = seqq.upper()
+        self.seq2 = self.seq.replace('T', 'U')
+        self.basecomp = {'A':'T', 'G':'C', 'T':'A', 'C':'G'}
+        self.letters = list()
+        self.trans_seq = {'A':'U', 'G':'C', 'U':'A', 'C':'G'}
+
+    def reverse_complement(self):
+        letters = [self.basecomp[base] for base in self.seq]
+        return ''.join(letters)
```

Comment:
You return the rigth complement. The reverse complement of a DNA sequence is formed by reversing the letters, interchanging A and T and interchanging C and G. Thus the reverse complement of 5'ACCTGAG 3' is 5' CTCAGGT 3'. Your return 3' TGGACTC 5'
 https://www.bx.psu.edu/old/courses/bx-fall08/definitions.html

Reply:
I suppose it will fix the problem
```suggestion
        self.seq = self.seq[::-1]
```

Reply:
I suppose '[::-1]' will fix the problem